### ISSUE TITLE
Implement accessibility nodes to handle focus concerns

### ISSUE THREAD

**Issue by rsubtil** [retrohub-org/retrohub#213]
Design for accessibility should be a complement to the current node structure, not requiring a separate project. One way to achieve this is to add a special node under focusable objects that "hijacks" focus settings when accessibility options are enabled.
